platforms/quorum/configuration/roles/helm_component/templates/tessera.tpl

```diff
@@ -44,16 +45,16 @@ spec:
       lock: {{ peer.lock | lower }}
       ports:
         rpc: {{ peer.rpc.port }}
-{% if network.config.consensus == 'raft' %}
```

Comment:
Why is this IF removed if raft is not needed, remove it entirely

Reply:
removed.